Cargo.lock

```diff
@@ -66,7 +90,7 @@ name = "actix-diesel"
 version = "0.3.1-alpha.0"
 source = "git+https://github.com/frol/actix-diesel?branch=actix-0.11-beta.2#7066a04ccce15b20940d926758837c99954765cc"
 dependencies = [
- "actix",
+ "actix 0.11.0-beta.2",
```

Comment:
Update actix-diesel commit hash to `3a001986c89dfabfc3c448d8bae28525101b4992` (I have just created that branch) to drop actix 0.11 from the dependencies